website/docs/migrating-from-docker/managing-docker-compatibility.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 5
+title: Managing docker compatibility
+description: Covers configurations available to manage docker compatibility
+keywords: [podman desktop, podman, docker, compatibility]
+tags: [managing-docker-compatibility]
+---
+
+# Managing docker compatibility
+
+With Podman Desktop, you can configure a Docker-compatible environment to run your Docker applications on a Podman engine, rather than on a Docker engine.
+
+As a developer, you can:
+
+- Set the Docker compatibility mode to emulate a Docker API socket connection to the `var/run/docker.sock` default path.
+- Use all Docker commands with Podman. For example, you can use the `podman run` command in place of `docker run` command to start a container.
```

Comment:
Whilst this is true, this is not what what this compatibility mode is providing. You can do it out of the box with Podman. 

What this compatibility mode does is that when you run `docker run` commands it will use the Podman engine to run your containers.